src/java/com/sarality/app/view/dialog/BaseDialogFragment.java

```diff
@@ -27,8 +17,8 @@
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState) {
     AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-    getActionManager().setup(builder);
     configure(builder);
+    getActionManager().setup(builder);
```

Comment:
Ideally the extension class should not be setting the Actions, they should be set on the ActionManager itself.

Even if you were not using the ActionManager and setup was called, it'll be a no-op since it sets up the Button actions only when they are not null.

You can check this in, but I'd like to understand what the problem is here so that we can solve it properly.